src/abstract-authenticated-command.ts

```diff
@@ -15,7 +15,18 @@ export default abstract class AbstractAuthenticatedCommand extends AbstractComma
     this.authenticator = authenticator;
   }
 
-  async checkAuthentication() {
+  async run(): Promise<void> {
```

Comment:
Is this really related to the flags?

Reply:
This is just some reordering, I can put it in another PR if you want.